dispatcher_plugin_nb2workflow/dataserver_dispatcher.py

```diff
@@ -194,43 +198,12 @@ def run_query(self,
                     except_message = resroot['exceptions'][0]['ename']+': '+res.json()['data']['exceptions'][0]['evalue']
                 else:
                     except_message = res.json()['exceptions'][0]
-
```

Comment:
this is not a change described in the PR title

Reply:
it was how it was when I created the PR first, then the logic we decided to implement changed (and I don't exclude this can change again)

Reply:
Volodymyr mean that removal of this block producing "download notebook" link is unrelated. I will add to the name